e2e/test/scenarios/question/question-management.cy.spec.js

```diff
@@ -178,19 +180,100 @@ describe("managing question from the question's details sidebar", () => {
               cy.findByText("This question has been archived");
             });
 
-            it("should be able to add question to dashboard", () => {
-              openQuestionActions();
-              cy.findByTestId("add-to-dashboard-button").click();
+            describe("Add to Dashboard", () => {
+              it("should be able to add question to dashboard", () => {
+                openQuestionActions();
+                cy.findByTestId("add-to-dashboard-button").click();
+
+                modal().as("modal").findByText("Orders in a dashboard").click();
+
+                cy.get("@modal").should("not.exist");
+                // By default, the dashboard contains one question
+                // After we add a new one, we check there are two questions now
+                cy.get(".DashCard").should("have.length", 2);
+              });
+
+              it("should preselect the most recently visited dashboard", () => {
```

Comment:
I don't think it's needed to run this test (and the next one) for different users. Can we just run it for the normal user?

Reply:
AFAIK test plan assumed we needed to cover permissions, that's why I checked several users. But it totally makes sense to me to test only normal user as permissions are covered by another test now